lib/chatops/controller.rb

```diff
@@ -27,6 +27,8 @@ def list
     end
 
     def process(*args)
+      params.merge! json_body
```

Comment:
:art: With Strong Parameters in Rails 5, this `merge!` would work but the parameters would be unpermitted by default.  Example:

```
# Not Permitted (as implemented above)
ActionController::Parameters.new(a: 1).merge! b: 2
# => <ActionController::Parameters {"a"=>1, "b"=>2} permitted: false>

# Permitted (what might need to be fixed)
ActionController::Parameters.new(a: 1).permit(:a).merge! b: 2
# => <ActionController::Parameters {"a"=>1, "b"=>2} permitted: true>

```